simphony_ui/tests/test_cfd_dem.py

```diff
@@ -0,0 +1,25 @@
+"""
+Tests
+"""
+
+import unittest
+import simphony_ui.couple_CFDDEM as cfd_dem
```

Comment:
General python convention. We order the imports as system, then libraries, then own package. Some people also want them alphabetically ordered.